tests/steps/application_v2_steps.py

```diff
@@ -406,16 +408,15 @@ def remember_app_id(context):
 def wait_for_app_txn_confirm(context):
     sp = context.app_acl.suggested_params()
     last_round = sp.first
+    send_zero_transactions(context, 3)
```

Comment:
Curious on how the number 3 was arrived at.

Reply:
In the line below, it checks the status after block `last_round + 2`, so I sent 3 transactions (advances 3 blocks in dev mode) for good measure!